components/ArticlesList.js

```diff
@@ -0,0 +1,77 @@
+import {
+  StyleSheet,
+  FlatList,
+  Text,
+  View,
+  TouchableOpacity,
+} from "react-native";
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { shouldUseActivityState } from "react-native-screens";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    paddingTop: 22,
+  },
+  item: {
+    padding: 10,
+    fontSize: 30,
+    height: 44,
+  },
+  teaser: {
+    padding: 10,
+    fontSize: 20,
+  },
+  published: {
+    padding: 10,
+    fontSize: 15,
+  },
+  card: {
+    border: "solid #dbdbdb",
+    margin: 5,
+    borderRadius: 10,
+    backgroundColor: "#e8e8e8",
+  },
+});
+
+const ArticlesList = ({ navigation }) => {
+  const [articles, setArticles] = useState([]);
+
+  const fetchArticles = async () => {
+    try {
+      const response = await axios.get(
+        "https://yesterdays-news-api.herokuapp.com/api/articles"
+      );
+      setArticles(response.data.articles);
+      debugger;
+    } catch (error) {}
```

Comment:
You can display a message on the view if we get an error from the api call, kinda like how we do it on the client application